runtime/util/vmargs.c

```diff
@@ -1496,11 +1537,20 @@ addLauncherArgs(J9PortLibrary * portLib, JavaVMInitArgs *launcherArgs, UDATA lau
 		}
 
 		if (TRUE == convertEncoding) {
-			optArg = newJavaVMArgInfo(vmArgumentsList, transcodedString, ARG_MEMORY_ALLOCATION|consumableFlag);
+			if(!isGeneratedArgument(transcodedString)) {
+				/* We did not find the argument in the table of Java Launcher Args, so we treat it as a user arg. */
+				optArg = newJavaVMArgInfo(vmArgumentsList, transcodedString, ARG_MEMORY_ALLOCATION|CONSUMABLE_ARG|USER_ARG);
+			} else {
+				optArg = newJavaVMArgInfo(vmArgumentsList, transcodedString, ARG_MEMORY_ALLOCATION|CONSUMABLE_ARG);
```

Comment:
Duplicated code: use the ternary operator to add either `CONSUMABLE_ARG|USER_ARG` or `CONSUMABLE_ARG`.

Reply:
Ah sorry, must've missed this